clients/tabby-chat-panel/src/index.ts

```diff
@@ -33,7 +34,7 @@ export interface ChatMessage {
 }
 
 export function createClient(target: HTMLIFrameElement) {
```

Comment:
```suggestion
export function createClient(target: HTMLIFrameElement): Api {
```

Reply:
updated, I actually add ServerAPI and ClientAPI